e2e/cypress/integration/enterprise/system_console/session_length_spec.js

```diff
@@ -29,7 +29,7 @@ const waitUntilConfigSave = () => {
 const saveConfig = (waitUntilConfigSaved = true, clickConfirmationButton = false) => {
     // # Save if possible (if previous test ended abruptly all permissions may already be enabled)
     cy.get('#saveSetting').then((btn) => {
-        if (!btn.disabled) {
```

Comment:
fun fact - `btn.disabled` always returns `undefined` so all of these `!btn.disabled` checks were doing always returning true - effectively doing nothing!